content/docs/guides/crosswalk/kubernetes/introduction/00-prerequisites.md

```diff
@@ -1,15 +1,14 @@
 ---
 title: Prerequisites
 
-aliases: ["/docs/quickstart/k8s-the-prod-way/prerequisites/"]
+aliases: ["/docs/guides/crosswalk/kubernetes/prerequisites/"]
 ---
 
 This tutorial leverages [Pulumi][pulumi] and [Node.js][nodejs] to provision and configure
```

Comment:
I assume Python is in scope?

Reply:
IIUC Python was not in scope given the time constraint. Also, managing EKS is one of the areas that is only available today in TS.

I'll rephrase this to not tie ourselves to a given language, and make a note for us to follow up with python examples in any applicable areas.

Reply:
At least for everything at the app layer - it would be nice to have examples using langchooser and be in both Node.js and Python.  I assume for most of the infra layer we could also do this - though EKS/AWSX may make things difficult there.  Getting a first pass done with Node.js makes sense - but I think structurally we should think about this as multi-language documentation.